src/content/changes/highlights/2021-07-16.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-07-16T10:00:00
+title: Highlights for the week ending July 16, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Managed apps
+
+[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Potentially Breaking change**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable it please check the details in the [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/).
+
+[Prometheus Operator v0.9.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.9.0/) uses `giantswarm/config` to generate managed configuration. This is part of a larger [story](https://github.com/giantswarm/roadmap/issues/121) to automate the configuration management of installations.
+
+## User interfaces
+
+In [kubectl gs v1.32.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.32.0/), Catalog CRD replaces AppCatalog CRD, enabling organization-wide (rather than 'global'), namespace-scoped, app catalogs. It also adds templating using CAPA (Cluster API Provider AWS) upstream templates for clusters in AWS v20.0.0, in preparation for creating CAPI clusters on Giant Swarm.
```

Comment:
@marians , this one is new. Can you check if the CAPA stuff is correct? I am not sure.`

Reply:
This

> It also adds templating using CAPA (Cluster API Provider AWS) upstream templates for clusters in AWS v20.0.0, in preparation for creating CAPI clusters on Giant Swarm.

is something Team Firecracker preferred not to highlight, as customers can't use v20 yet.

Reply:
OK will delete thank you!